hierarc/Likelihood/hierarchy_likelihood.py

```diff
@@ -290,11 +310,20 @@ def log_likelihood_single(
             if self._gamma_in_array is not None and self._log_m2l_array is not None:
                 lnlikelihood -= (
                     self._gamma_in_prior_mean - scaling_param_array[-2]
-                ) ** 2 / (2 * self._gamma_in_prior_std**2)
+                ) ** 2 / (2 * self._gamma_in_prior_std**2) + np.log(
+                    self._gamma_in_prior_std * (2 * np.pi) ** 0.5
+                )
             elif self._gamma_in_array is not None and self._log_m2l_array is None:
                 lnlikelihood -= (
                     self._gamma_in_prior_mean - scaling_param_array[-1]
-                ) ** 2 / (2 * self._gamma_in_prior_std**2)
+                ) ** 2 / (2 * self._gamma_in_prior_std**2) + np.log(
+                    self._gamma_in_prior_std * (2 * np.pi) ** 0.5
+                )
+
+        if self._kappa_marginalize_pdf is True and self._kappa_pdf_trunc is not None:
```

Comment:
can you describe in more detail of what's happening here?

Reply:
Sure!  The idea is that I want to somehow fold in prior information from individual systems into hierarc.  E.g., I want to somehow use the individual kappa_ext PDFs in the hierarchical process (since I'm now parameterizing and sampling from a population sample instead from individual distributions).  So now, I add to an individual system's lnlikelihood such that, for a given sampled kappa_ext (sampled from the population level), I add how likely it is to have been sampled from the individual system distribution.

I do the same process for gamma_in as well (except its in the form of a mean prior and std instead of a PDF)!